Mapify/SceneInitializers/GameContent/VanillaAssetSetup.cs

```diff
@@ -12,23 +13,35 @@ public override void Run()
         {
             foreach (VanillaObject vanillaObject in Object.FindObjectsOfType<VanillaObject>())
             {
-                GameObject go = vanillaObject.Replace(keepChildren: vanillaObject.keepChildren, rotationOffset: vanillaObject.rotationOffset);
+                //only these belong in the gamecontent scene
+                var yes = new List<VanillaAsset>
+                {
+                    VanillaAsset.CareerManager,
+                    VanillaAsset.JobValidator,
+                    VanillaAsset.TrashCan,
+                    VanillaAsset.Dumpster,
+                    VanillaAsset.LostAndFoundShed,
+                    VanillaAsset.WarehouseMachine,
+                    VanillaAsset.PlayerHouse,
+                    VanillaAsset.PitStopStationCoal1,
+                    VanillaAsset.PitStopStationCoal2,
+                    VanillaAsset.PitStopStationWater1,
+                    VanillaAsset.PitStopStationWater2,
+                    VanillaAsset.StationOffice1,
+                    VanillaAsset.StationOffice2,
+                    VanillaAsset.StationOffice3,
+                    VanillaAsset.StationOffice4,
+                    VanillaAsset.StationOffice5,
+                    VanillaAsset.StationOffice6,
+                    VanillaAsset.StationOffice7
+                };
 
-                switch (vanillaObject.asset)
+                if (!yes.Contains(vanillaObject.asset))
```

Comment:
We would've been better with the switch here rather than iterating though a list for each item. This is O(n*m) compared to O(n), and is harder to read.

Reply:
You are right. Updated.